bpf/lib/dbg.h

```diff
@@ -148,34 +149,28 @@ struct debug_msg {
 static __always_inline void cilium_dbg(struct __ctx_buff *ctx, __u8 type,
 				       __u32 arg1, __u32 arg2)
 {
-	__u32 hash = get_hash_recalc(ctx);
```

Comment:
All the `get_hash_recalc()` removal should be moved to first commit?

Reply:
I moved it into the __notify_common_hdr(), I'm also fine to remove the macro entirely, no strong opinion.

Reply:
Ah right it's moved, not removed. Ok it's fine as is then, sorry.